src/words/ast/INodeMovesPredicate.java

```diff
@@ -10,7 +15,22 @@ public INodeMovesPredicate(Object... children) {
 
 	@Override
 	public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException {
-		// TODO
-		throw new AssertionError("Not yet implemented");
+		ASTValue className = children.get(0).eval(environment);
```

Comment:
Ah, I didn't realize that we still allowed listeners to be attached to individual objects.  

Ooh, this changes a lot. I will take care of this later tonight.